sqlmesh/core/model/meta.py

```diff
@@ -108,16 +108,33 @@ def extract(v: exp.Expression) -> t.Tuple[str, t.Dict[str, exp.Expression]]:
         if isinstance(v, exp.Expression):
             return [extract(v)]
         if isinstance(v, list):
-            return [
-                (
-                    entry[0].lower(),
-                    {
-                        key: d.parse(value)[0] if isinstance(value, str) else value
-                        for key, value in entry[1].items()
-                    },
-                )
-                for entry in v
-            ]
+            audits = []
+
+            for entry in v:
+                if isinstance(entry, tuple):
+                    audits.append((
+                        entry[0].lower(),
+                        {
+                            key: d.parse_one(value) if isinstance(value, str) else value
+                            for key, value in entry[1].items()
+                        },
+                    ))
+                elif isinstance(entry, dict):
+                    if len(entry) != 1:
```

Comment:
Not sure I understand what the format is going to look like. Should we add a test?